internal/node/npm_package_bin.bzl

```diff
@@ -78,6 +78,7 @@ def _impl(ctx):
         stdout = ctx.outputs.stdout,
         stderr = ctx.outputs.stderr,
         exit_code_out = ctx.outputs.exit_code_out,
+        execution_requirements = {tag: "" for tag in ctx.attr.tags},
```

Comment:
Hmm it would be nice to have some usage of the feature, as a regression test and also so others can see this workaround for `no-remote-exec` in action. I understand that's tricky, we don't have a 90k-input-count action to reproduce the same problem you had. Maybe we need a test that runs `bazel aquery` or something...

Also are there any docs about how remote execution honors certain tags and how they should be propagated via the execution requirements? I'd like a doc linked here so we can understand why it's like this.

Reply:
There were 2 issues on my case:
1. RBE was throwing that error when trying to build a react project. Bazel targets have been copied from [here](https://github.com/bazelbuild/rules_nodejs/tree/master/examples/create-react-app)
2. In order to not hit this RBE bug, I tried to force the target to always compile locally via "no-remote-exec" tag. This didn't worked because the tag was not propagated to this action

Here is the bazel issue about tags not being propagated: https://github.com/bazelbuild/rules_nodejs/tree/master/examples/create-react-app

Reply:
That last link looks wrong, do you have a right one?
From https://github.com/bazelbuild/rules_go/issues/2534#issuecomment-642024716 it seems like `--experimental_allow_tags_propagation` is what you need, then it works everywhere not just this one spot